extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -92,11 +92,11 @@ public static final class DestinationInfo {
 
         private final String outputPath;
         private final Table inputTable;
-        private final Map<String, Map<?, long[]>> columnNameToGroupToRange;
+        private final Map<String, Map<?, RowSet>> columnNameToGroupToRange;
```

Comment:
I understand why DHE changed this. This is unused code for us, it's tied closely to `MergeData`, and should probably just be deleted.

Reply:
Yep, see the issue for this, there is a lot of dead code in here, but I haven't been as aggressive about removing things, esp stuff that I understood (i.e. "I interpreted, possibly wrongly, from discussions with andy and/or cristian") might be important later. Someone who knows the code at all, and has some idea of future plans/intentions should take a closer look at how deeply we can prune apparently unused code.

Reply:
This is a case where my confidence level in our deletion is high. If this became important, we would be porting `MergeData` , and could bring it back over then. We'd be more likely to write our own merge tool that leverages parquet functionality better.